Providers/Scripts/2.6x-2.7x/Scripts/Tests/test_nxProviders.py

```diff
@@ -1998,6 +1998,18 @@ def testInventoryMarshall(self):
         self.assertTrue(r[0] == 0,"Inventory_Marshall('*', " + self.provider + "provider, False,'')  should return == 0")
         print repr(r[1])
 
+    def testInventoryMarshallControllerWildcard(self):
+        r=nxService.Inventory_Marshall('*', '*', False,'')
+        self.assertTrue(r[0] == 0,"Inventory_Marshall('*', " + self.provider + "provider, False,'')  should return == 0")
```

Comment:
The error message looks wrong. We are calling `Inventory_Marshall('*', '*', False,'')` not `Inventory_Marshall('*', 'nxProviderprovider', False,'')`